src/pages/App.js

```diff
@@ -22,26 +22,32 @@ function App() {
   );
 
   return (
-    <div className="flex justify-center">
-      <div className="-mt-6 max-w-6xl">
-        {/* Negative margin sized to height of fileSelect –  moves select moved up into margins, aligns titles */}
-        <FileSelect files={files} onChange={changeDataSource} />
-        <h1 className="font-sans font-bold text-4xl">Coverage Overview</h1>
-        <p className="text-sm text-gray-600 pb-2">Select a category to analyze it in finer detail</p>
-        <div className="flex flex-row pb-5 gap-5 items-stretch">
-          <MainVisualization
-            coverageData={coverageData}
-            selectedSection={selectedSection}
-            setSelectedSection={setSelectedSection}
-          />
-          <Rankings coverageData={coverageData} />
-        </div>
-        <h2 className="font-sans font-semibold text-2xl">Analysis</h2>
-        <p className="text-sm text-gray-600">Fine tune your analysis through your selection of subcategories</p>
-        <div className="flex flex-row gap-5 items-start">
-          <SubcategoryTable selectedSection={selectedSection} coverageData={coverageData} />
-          <Longitudinal selectedSection={selectedSection} coverageData={coverageData} />
-        </div>
+    <div className="-mt-6 max-w-6xl">
+      {/* Negative margin sized to height of fileSelect –  moves select moved up into margins, aligns titles */}
+      <FileSelect files={files} onChange={changeDataSource} />
+      <h1 className="font-sans font-bold text-4xl">Coverage Overview</h1>
+      <p className="text-sm text-gray-600 pb-2">Select a category to analyze it in finer detail</p>
+      <div className="flex flex-row max-md:flex-wrap pb-5 gap-5 items-stretch">
+        <MainVisualization
+          coverageData={coverageData}
+          selectedSection={selectedSection}
+          setSelectedSection={setSelectedSection}
+        />
+        <Rankings coverageData={coverageData} />
+      </div>
+      <h2 className="font-sans font-semibold text-2xl">Analysis</h2>
+      <p className="text-sm text-gray-600">Fine tune your analysis through your selection of subcategories</p>
+      <div className="flex flex-row max-md:flex-wrap gap-5 items-start">
+        <SubcategoryTable
+          className="h-[500px] lg:w-1/2 max-md:grow"
+          selectedSection={selectedSection}
+          coverageData={coverageData}
+        />
+        <Longitudinal
+          className="h-[500px] lg:w-1/2 max-md:grow"
```

Comment:
```suggestion
          className="h-[500px] lg:w-1/2 max-lg:w-full"
```
Wrapping on the lg breakpoint looks a bit better
Also think we get slightly more reasonable css if we define with here instead of a flex property